manifest.json

```diff
@@ -1,11 +1,14 @@
 {
-  "name": "ts-template",
-  "description": "ts-template for UbiquityOS plugins.",
-  "ubiquity:listeners": ["issue_comment.created", "issue_comment.deleted", "issue_comment.edited"],
+  "name": "scan-contributions",
+  "description": "UbiquityOS plugin for scanning contributions based on issues/pulls.",
+  "ubiquity:listeners": [
+    "issue_comment.created",
+    "issue_comment.edited"
+  ],
   "commands": {
     "command1": {
```

Comment:
I think the kernel is supposed to ignore `/scan-contribution` command because the command is not right here. I don't know how it is going from kernel to plugin with this. 
![image](https://github.com/user-attachments/assets/7aa134ef-3723-4394-92fa-2f8d28e1285a)

Reply:
Set a webhook that points directly to worker URL.